osu.Game/Online/API/DummyAPIAccess.cs

```diff
@@ -63,12 +63,13 @@ public DummyAPIAccess()
 
         public virtual void Queue(APIRequest request)
         {
-            if (HandleRequest?.Invoke(request) != true)
+            Schedule(() =>
```

Comment:
To confirm: `Queue()` is the only case where the invocation of `HandleRequest` was scheduled, because it is presumed that `Perform()` and `PerformAsync()` should never be called by a disposed drawable?

Reply:
> To confirm: `Queue()` is the only case where the invocation of `HandleRequest` was scheduled, because it is presumed that `Perform()` and `PerformAsync()` should never be called by a disposed drawable?

Basically. See the alternative version where I also scheduled those, and the ratification for why that version doesn't work. I would have liked to do it everywhere but I don't really want to go through every update-thread-blocking beatmap import to make that work.